content/en/apps/reference/contact-page.md

```diff
@@ -34,7 +34,7 @@ Each field that can be shown on a contact's profile is defined as an object in t
 | `label` | `string` | A translation key which is shown with the field. | yes |
 | `icon` | `string` | The name of the icon to display beside this field, as defined through the Configuration > Icons page. | no |
 | `value` | `string` | The value shown for the field. | yes |
-| `filter` | `string` | The display filter to apply to the value, eg: `{ value: '2005-10-09', filter: 'age' }` will render as "11 years". Common filters are: `age`, `phone`, `weeksPregnant`, `relativeDate`, `relativeDay`, `fullDate`, `simpleDate`, `simpleDateTime`, `lineage`, `resourceIcon`, `translate`. For the complete list of filters, and more details on what each does, check out the code in [`medic/webapp/src/js/filters` dir](https://github.com/medic/medic/tree/master/webapp/src/js/filters). | no |
+| `filter` | `string` | The display filter to apply to the value, eg: `{ value: '2005-10-09', filter: 'age' }` will render as "11 years". Common filters are: `age`, `phone`, `weeksPregnant`, `relativeDate`, `relativeDay`, `fullDate`, `simpleDate`, `simpleDateTime`, `lineage`, `resourceIcon`, `translate`. For the complete list of filters, and more details on what each does, check out the code in [`medic/webapp/src/js/filters` dir](https://github.com/medic/cht-core/tree/3.10.x/webapp/src/js/filters). | no |
```

Comment:
Was the `3.10.x` intentional here? Presumably we'd need to update this link over time as new versions are released.

Reply:
Sorry, the note about this is hidden in the commit message: "Temporary fix until filters documented fully in docs."

The code got completely refactored in 3.11, so we don't have somewhere easy to point to. We should properly document the filters individually in these docs, rather than link to code anyhow, but in the meantime linking to 3.10.x would be fine.

Reply:
Gotcha.

For the refactor, these are now "pipes" and the code has been moved/refactored to the `pipes` directory for 3.11+

```suggestion
| `filter` | `string` | The display filter to apply to the value, eg: `{ value: '2005-10-09', filter: 'age' }` will render as "11 years". Common filters are: `age`, `phone`, `weeksPregnant`, `relativeDate`, `relativeDay`, `fullDate`, `simpleDate`, `simpleDateTime`, `lineage`, `resourceIcon`, `translate`. For the complete list of filters, and more details on what each does, check out the code in [`webapp/src/ts/pipes` dir](https://github.com/medic/cht-core/tree/3.11.x/webapp/src/ts/pipes). | no |
```